src/main/resources/rulesets/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+"http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <!-- These suppressions define which files to be suppressed for which checks. -->
+    <suppress files=".+[\\/]internal[\\/].+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
+    <suppress files=".+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck" />
+    <suppress files=".+Impl\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
+    <!-- The WiFi led binding contains test packages that shouldn't be included in the binary build -->
+    <!-- The CometVisu UI contains files in the resources folder that are used only during the build -->
+    <suppress files=".+org.openhab.binding.wifiled.+|.+org.openhab.ui.cometvisu.+" checks="BuildPropertiesCheck"/>
+    <!-- Homematic and Tellstick bindings are creating and configuring things dynamically, they will log false positives for unused configuration -->
+    <!-- IO and Voice bundles have specific use cases - they use only the config .xml files and will log false positives as well -->
+    <suppress files=".+org.openhab.binding.homematic.+|.+org.openhab.binding.tellstick.+|.+org.openhab.io.+|.+org.openhab.voice.+" checks="EshInfXmlCheck"/>
+    <!-- All generated files will skip the author tag check -->
+    <suppress files=".+[\\/]gen[\\/].+\.java" checks="AuthorTagCheck"/>
+    <!-- Some checks will be supressed for test bundles -->
+    <suppress files=".+.test[\\/].+" checks="RequireBundleCheck"/>
+    <suppress files=".+org.openhab.tools.analysis.report.ReportUtility.java+|.+org.openhab.tools.analysis.findbugs.CustomClassNameLengthDetector.java+" checks="RegexpHeaderCheck"/>
```

Comment:
We don`t need this one here, it will be only in the project specific suppression file, remove it